sdk/signalr/Microsoft.Azure.Management.SignalR/src/Microsoft.Azure.Management.SignalR.csproj

```
@@ -6,10 +6,10 @@
 	<PropertyGroup>
 		<PackageId>Microsoft.Azure.Management.SignalR</PackageId>
 		<Description>Provides management capabilities for Azure SignalR.</Description>
-		<Version>1.0.1</Version>
+		<Version>1.0.2</Version>
```

Suggest incrementing the minor version for feature addition. Update the patch version if it's bug-fixing only.
```suggestion
		<Version>1.1.0</Version>
```